config.example.wikimedia.yaml

```diff
@@ -53,6 +53,20 @@ spec_root: &spec_root
     # global domain
     /{domain:wikimedia.org}: *wikimedia.org
 
+    # A robots.txt to make sure that the content isn't indexed.
```

Comment:
Would be nice to add this to `config.test.yaml` as well and have a test for it, either via an `x-amples` stanza or a proper test under `test/features/`